src/emcpy/plots/create_map.py

```diff
@@ -0,0 +1,295 @@
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from map_tools import Domain, MapProjection
+
+
+__all__ = ['__init__', 'add_features', 'draw_data', 'add_title', 'add_xlabel',
```

Comment:
You don't need this line.
Add the following in `src/emcpy/plots/__init__.py`
```
from .create_map import CreateMap
```

You definitely should not put `__init__` here.
All the methods are private to the class, so you don't need them here.
The Class itself is public and that should be enough.